keps/sig-node/24-apparmor-ga/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: AppArmor graduation to General Availability (GA)
+kep-number: 24
+authors:
+  - "@saschagrunert"
+  - "@tallclair"
+owning-sig: sig-node
+participating-sigs:
+  - sig-api-machinery
+  - sig-auth
+status: implementable
+creation-date: 2020-01-10
+reviewers:
+  - TBD
+approvers:
+  - TBD
+prr-approvers:
+  - wojtek-t
+see-also:
+  - "/keps/sig-node/20190717-seccomp-ga.md"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.20"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.4"
+  beta: "v1.4"
+  stable: "v1.20"
```

Comment:
this missed 1.20 deadline, I'd expect 1.21

Reply:
Yes, changed to 1.21